reana.yaml

```diff
@@ -0,0 +1,33 @@
+inputs:
+  files:
+    - BuildFile.xml
+
+  directories:
+    - src
+    - data
+    - python
+    - configs
+
+workflow:
+  type: serial
+  specification:
+    steps:
+      - name: test
```

Comment:
I confirm that the workflow works well :+1: 

![AOD2NanoAODOutreachTool](https://user-images.githubusercontent.com/517546/220354829-03f68e70-d48b-4e41-aaf3-f9924cbbe4ac.png)

and produces ROOT file:

```console
$ reana-client ls -w AOD2NanoAODOutreachTool -h --filter name=output.root
NAME                                                 SIZE      LAST-MODIFIED
CMSSW_5_3_32/src/workspace/AOD2NanoAOD/output.root   8.7 GiB   2023-02-19T19:50:13
```

Two comments:

1. The step of the workflow is called "test". Perhaps a better name could be "data-production" or "test-data-production" or something?

2. The workflow produces a lot of files:
```console
$ reana-client ls -w AOD2NanoAODOutreachTool | wc -l
1049
```

Most are from scramming etc located here:

```console
$ reana-client ls -w AOD2NanoAODOutreachTool | grep CMSSW_5_3_32 | wc -l
923
```

It should not be necessary to keep them for long, so it may be useful to introduce a new [workspace file retention](https://blog.reana.io/posts/2022/workspace-file-retention-rules/) configuration in `reana.yaml`, something like:

```yaml
workspace:
  retention_days:
    CMSSW_5_3_32: 7
```

so that the `CMSSW_5_3_32` directory would be automatically deleted after 7 days.

P.S. Just an example to be adapted based on what you would wish to preserve for shorter/longer, e.g. remove only `CMSSW_5_3_32/.SCRAM`, etc.